integrationTests/CampaignStatsAfterDonationCollectedTest.php

```diff
@@ -140,8 +141,8 @@ private function runStatsUpdateCommand(): void
         $application = $this->buildMinimalApp($lockFactory);
         $command = new UpdateCampaignDonationStats(
             $this->getContainer()->get(CampaignRepository::class),
-            $this->getContainer()->get(CampaignStatisticsRepository::class),
             $this->getContainer()->get(EntityManagerInterface::class),
+            $this->getContainer()->get(MatchFundsService::class),
```

Comment:
not critical but I think we could just ask the container  for a `UpdateCampaignDonationStats` and let it reflect on how to construct it.